app/controllers/catalog_controller.rb

```diff
@@ -955,9 +957,7 @@ def service_template_list(scope, options = {})
       @gtl_buttons = %w[view_list view_tile]
       @gtl_small_tiles = true
       if role_allows?(:feature => 'svc_catalog_provision')
-        @row_button = {:label    => _("Order"),
-                       :function => "miqOrderService",
-                       :title    => _("Order this Service")} # Show a button instead of the checkbox
+        @row_button = true # Show a button instead of the checkbox
```

Comment:
You can make this a one-liner:
```suggestion
      # Show a button instead of the checkbox
      @row_button = true if role_allows?(:feature => 'svc_catalog_provision')
```